modules/monitoring-support-considerations.adoc

```diff
@@ -21,3 +21,4 @@ The Alertmanager configuration is deployed as a secret resource in the `openshif
 * *Installing custom Prometheus instances on {product-title}.*
 * *Enabling symptom based monitoring by using the `Probe` custom resource definition (CRD) in Prometheus Operator.*
 * *Modifying Alertmanager configurations by using the `AlertmanagerConfig` CRD in Prometheus Operator.*
+* Backward compatibility for metrics, recording rules, or alerting rules is not guaranteed.
```

Comment:
lgtm, only need to update its font，should be bold like others
see from https://github.com/openshift/openshift-docs/blob/91c2ee21a3c5fcbb5ade6d88ca8c85833d6bcf7f/modules/monitoring-support-considerations.adoc

Reply:
Thank you! Going to update this into a note instead of a bullet per Christian's comment in the Jira ticket. 👍